scripts/main.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable import/extensions */
+import Book from './dom.js';
+
+const myLibrary = [];
+window.changeStatus = (statusId) => {
+  const lib = JSON.parse(localStorage.getItem('myLibrary'));
+  const retrievedId = document.getElementById(`s${statusId}`);
+  retrievedId.className = 'bg-danger';
+  retrievedId.innerHTML = 'read';
+  lib[statusId].status = true;
+  localStorage.setItem('myLibrary', JSON.stringify(lib));
+};
+window.removeItem = (itemid) => {
+  const lib = JSON.parse(localStorage.getItem('myLibrary'));
+  const newlib = lib.filter(item => item.itemid !== item);
+  localStorage.setItem('myLibrary', JSON.stringify(newlib));
+  document.getElementById(itemid).parentNode.parentNode.remove();
```

Comment:
- [ ] `changeStatus` and `removeItem` can also use the `display` function after you update your book list.